packages/pages/src/components/analytics/hooks.ts

```diff
@@ -0,0 +1,38 @@
+import { useContext } from "react";
+import { AnalyticsContext } from "./context";
+import { AnalyticsMethods } from "./interfaces";
+
+/**
+ * Returns a service that can be used to report analytics events
+ */
```

Comment:
Anything that a user will use should have a line in the jsdoc that's `@public`

Reply:
yeah i was going to do a documentation pass once things had settled a bit